google-chart.html

```diff
@@ -384,6 +404,7 @@
             var chart = chartAndData[0];
             var data = chartAndData[1];
             google.visualization.events.addOneTimeListener(chart, 'ready', function() {
+              this._setImageUri(this.imageURI);
```

Comment:
Maybe try:

properties: {
  imageURI: {
   ...
    value: function() {
      if (!this._imageUri) {
        this._imageUri = this._chartObject.getImageURI();
        }
      return this._imageUri;
    } 
  }
}